Src/DependencyCollector/Shared.Tests/Implementation/ProfilerHttpProcessingTest.cs

```diff
@@ -115,6 +115,27 @@ public void RddTestHttpProcessingProfilerOnExceptionForGetResponse()
         }
 
         /// <summary>
+        /// Validates HttpProcessingProfiler sends correct telemetry including response code on calling OnExceptionForGetResponse for WebException.
+        /// </summary>
+        [TestMethod]
+        [Description("Validates HttpProcessingProfiler sends correct telemetry including response code on calling OnExceptionForGetResponse for WebException.")]
+        [Owner("cithomas")]
+        [TestCategory("CVT")]
+        public void RddTestHttpProcessingProfilerOnWebExceptionForGetResponse()
+        {
+            var request = WebRequest.Create(this.testUrl);
+            this.httpProcessingProfiler.OnBeginForGetResponse(request);
+            Thread.Sleep(this.sleepTimeMsecBetweenBeginAndEnd);
```

Comment:
I have no idea why there's a sleep here, it's what all the other tests for this class are doing. Maybe @abaranch knows?